### ISSUE TITLE
10-10EZ improve UX for "already enrolled"

### ISSUE THREAD

**Issue by Dene-human** [department-of-veterans-affairs/va.gov-team#28081]
![image (1).png](https://images.zenhubusercontent.com/605c95113f60cac14b50309b/055fbab3-bcf2-4ce0-90b2-2c52ae83ba04)![image.png](https://images.zenhubusercontent.com/605c95113f60cac14b50309b/0f48159b-b705-453e-a21d-7f9f07cb01f1)

**Comment by Dene-human** [comment]
As far as I know nothing has changed.

**Comment by longmd** [comment]
@hdjustice , Id love to bring this back toward the front of queue for an improvement. I noticed a similar issue when signed in as an LOA1 user today, too, (see screenshot below) and feel like its worth revisiting. Maybe repurpose this ticket to ensure the correct display of information for each use case:

1. Already enrolled or any status where you cannot apply again
2. Already applied and awaiting decision
3. logged in LOA1 (do they need to verify before completing the application)
4. logged in LOA3
5. Guest user

**Logged in LOA1 user:**
![screencapture-staging-va-gov-health-care-apply-application-introduction-2023-09-13-17_18_56](https://github.com/department-of-veterans-affairs/va.gov-team/assets/6738544/b7e4b1f0-a4d5-4f46-94b9-c979ba08fd39)

**Comment by hdjustice** [comment]
I agree.  I linked Dene's ticket to this as well and we can look at it as a whole scope of work.  With the EZR being online, it would make sense to revamp the already enrolled experience
@jessicastump for awareness

**Comment by jessicastump** [comment]
Were some of these UX concerns addressed with ticket #47707 introduction page variations? There is a page for this in the Sketch file which was a key resource used during the error audit. [Sketch file](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/p/0C3B7020-8169-4FF2-AB5C-AF52BC4E34C2/canvas)

Also, opened ticket #65014 to update [this page](https://www.sketch.com/s/da85cf44-4503-4e98-834e-ff068b242ef6/a/MyZyrxW) - the already enrolled page

Agree, looking at the use cases and possible paths through the EZ and EZR as a whole/epic would be good!

**Comment by hdjustice** [comment]
Revisit as part of the Form Status feature work (#95313 )
